package.json

```diff
@@ -6,29 +6,30 @@
   "scripts": {
     "dev": "astro dev",
     "start": "astro dev",
-    "build": "astro build",
-    "build:prod": "IS_PROD_BUILD=true astro build",
-    "preview": "astro preview",
+    "build": "BUILD_TARGET=staging astro build",
+    "preview": "BUILD_TARGET=staging astro preview",
+    "build:prod": "BUILD_TARGET=production astro build",
+    "preview:prod": "BUILD_TARGET=production astro preview",
```

Comment:
@taoeffect 
I made some changes to tidy up `astro.config.mjs` in [this commit](https://github.com/okTurtles/groupincome.org/pull/177/commits/34a716284605f01c4707c4679b2b2f60f6571fcd).

Would you prefer to set the commands as below? (the command defaulting to `prod` option and having a variant for `staging`):

```

"build": "BUILD_TARGET=production astro build",
"preview": "BUILD_TARGET=production astro preview",
"build:staging": "BUILD_TARGET=staging astro build",
"preview:staging": "BUILD_TARGET=staging astro preview"
```